tests/Browser/BlogManageTest.php

```diff
@@ -0,0 +1,562 @@
+<?php
+
+namespace Tests\Browser;
+
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+
+class BlogManageTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+
+    public function setUp() :void
+    {
+        parent::setUp();
+        $this->artisan('db:seed --class=UserTableTestAddUserSeeder');
+        $this->artisan('db:seed --class=BlogTableTestAddBlogSeeder');
+    }
+
+    /**
+     * Get data from cogfig to test.
+     * @return array
+     */
+    public function getDataTest()
+    {
+        return config('constants');
+    }
+    
+    /**
+     * Test login blog with user can see blog.
+     * case 1.1: Test screen Blog list
+     *
+     * @return void
+     */
+    public function testLoginBlog()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->assertPathIs($data['path']['blog']);
+        });
+    }
+
+    /**
+     * Test popup create blog with user can see blog.
+     * case 1.2: Test popup Create blog
+     *
+     * @return void
+     */
+    public function testShowPopupCreateBlog()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->assertPresent('#createModal');
+        });
+    }
+
+     /**
+     * Test screen Edit blog by user.
+     * case 1.3: Edit blog: Test screen Edit blog by user
+     *
+     * @return void
+     */
+    public function testScreenEditBlog()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->click('#blog-edit-2')
+                    ->assertPathIs('/blog/2/edit');
```

Comment:
this case in Test Case have alot of expected , why in here only have 1

Reply:
I fixed